### PR TITLE
(docs) Convert rst to markdown to make contributions easier

### PR DESCRIPTION
**Pull request by ghoneycutt**

### CODE REVIEW

**Comment by ghoneycutt** [comment]
CI is broken which is not related to this simple PR.

**Comment by ghoneycutt** [comment]
@baurmatt could you have a look at this one please

File: README.md [hunk 1]
```diff
@@ -0,0 +1,642 @@
...
+
+> puppet module install puppet-puppetboard
+
+To see how to get it working with EL7 check out these [docs](https://github.com/voxpupuli/puppetboard/blob/master/docs/EL7.md).
+
+#### Manual
+
+To install it simply issue the following command:
+
+``` {.sourceCode .bash}
```
Comment (baurmatt) [inline README.md]: >\`\`\`bash

is enough.

`{.sourceCode ...}` shouldn't be needed.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,642 @@
...
+
+where X and Y are replaced by your major and minor python versions.
+
+You will need this path in order to configure your HTTPD and WSGI-capable application server.
+
+#### Packages
+
+Native packages for your operating system will be provided in the near future.
+
+<table>
```
Comment (baurmatt) [inline README.md]: Shouldn't this be a native markdown table? -> https://help.github.com/articles/organizing-information-with-tables/

File: README.md (hunk 1 above)
Comment (baurmatt) [reply README.md]: There are a lot of other code blocks which might need that change.

File: docs/EL7.md [hunk 3]
```diff
@@ -0,0 +1,203 @@
+# Steps to get this working with EL7
```
Comment (bastelfreak) [inline docs/EL7.md]: Awesome jobs, thanks for this tutorial!

**Comment by ghoneycutt** [comment]
Rebased.

File: README.md (hunk 2 above)
Comment (ghoneycutt) [reply README.md]: That would be a great contribution! This PR is simply meant to get us to markdown in the simplest way possible so further contributions can be done easily. The motivation is from spending entirely too much time trying to figure out RST where that took longer than the fix I was implementing.  The conversion is done using pandoc. https://pandoc.org/

**Comment by ghoneycutt** [comment]
@bastelfreak rebased and ready for merge. Thanks!

**Comment by bastelfreak** [comment]
thanks @ghoneycutt !

**Pull request merged**